lix/source-code/client/src/helpers.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, it, expect } from "vitest"
+import { parseLixUri } from "./helpers.js"
+
+// TODO: add local uris when implemented: "file://repo"
+
+describe("parse lix uris", () => {
+	it("parses github uris correctly", () => {
+		const parseResult = parseLixUri("https://github.com/inlang/monorepo")
+
+		expect(parseResult).toStrictEqual({
+			protocol: "https:",
+			lixHost: "",
+			namespace: "",
+			repoHost: "github.com",
+			owner: "inlang",
+			repoName: "monorepo",
+		})
+	})
```

Comment:
that tests contradicts the last test "should throw an misformed uri" where opening a github link is not possible.

Reply:
no that test checks if missing repo names are catched. i added explicit error test names to avoid this confusion.